packages/core/src/Stores/contract-trade-store.js

```diff
@@ -238,8 +240,8 @@ export default class ContractTradeStore extends BaseStore {
                 const trade_type_is_supported = trade_types.indexOf(info.contract_type) !== -1;
                 // both high_low & rise_fall have the same contract_types in POC response
                 // entry_spot=barrier means it is rise_fall contract (blame the api)
-                if (trade_type_is_supported && info.barrier && info.entry_tick && is_call_put) {
-                    if (`${+info.entry_tick}` === `${+info.barrier}`) {
+                if (trade_type_is_supported && is_call_put && ((info.barrier && info.entry_tick) || info.shortcode)) {
```

Comment:
Added additional check as sometimes first `info` doesn't include `barrier` and `entry_tick` fields, but includes `shortcode`. That caused an issue when user simultaneously open tabs with Rise/Fall and Higher/Lower contracts